src/app/component/general/eco-news/eco-news-detail/eco-news-detail.component.css

```diff
@@ -0,0 +1,102 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Montserrat', sans-serif;
```

Comment:
General: according to mock for title you should use font family 'Montserrat' and for another elements 'OpenSans'. Please double check it with mockup and fix. And remove *, it's bad practice

Reply:
Fixed